<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageMPSettingsTeamSpecific</class>
 <widget class="QWidget" name="PageMPSettingsTeamSpecific">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>298</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,0,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_72">
     <property name="text">
      <string>Player Species</string>
     </property>
     <property name="buddy">
      <cstring>fieldSpecies</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="label_71">
     <property name="text">
      <string>Initial Designator</string>
     </property>
     <property name="buddy">
      <cstring>fieldInitialDesignator</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_51">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="fieldEnableColorPrimary">
     <property name="toolTip">
      <string>If this box is not checked, a game-engine-default color will be used.</string>
     </property>
     <property name="text">
      <string>Set Primary Color</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="fieldName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Team Name Here</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_73">
     <property name="text">
      <string>Fireteam Count</string>
     </property>
     <property name="buddy">
      <cstring>fieldFireteamCount</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="fieldEnableColorText">
     <property name="toolTip">
      <string>If this box is not checked, a game-engine-default color will be used.</string>
     </property>
     <property name="text">
      <string>Set Tertiary Color</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="fieldEnableColorSecondary">
     <property name="toolTip">
      <string>If this box is not checked, a game-engine-default color will be used.</string>
     </property>
     <property name="text">
      <string>Set Secondary Color</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="buttonName">
     <property name="text">
      <string>Edit...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QCheckBox" name="fieldEnabled">
     <property name="text">
      <string>Enabled</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QPushButton" name="fieldButtonColorPrimary">
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="toolTip">
      <string>Click here to open a color picker. This is the main team color, used for armor and objectives. Very dark colors will appear differently in-game due to how the shaders work.</string>
     </property>
     <property name="accessibleName">
      <string>Edit Primary Color</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QPushButton" name="fieldButtonColorSecondary">
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="toolTip">
      <string>Click here to open a color picker.</string>
     </property>
     <property name="accessibleName">
      <string>Edit Secondary Color</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QPushButton" name="fieldButtonColorText">
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="toolTip">
      <string>Click here to open a color picker.</string>
     </property>
     <property name="accessibleName">
      <string>Edit Text Color</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QComboBox" name="fieldSpecies">
     <item>
      <property name="text">
       <string>Spartan</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Elite</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="2" colspan="2">
    <widget class="QSpinBox" name="fieldFireteamCount">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>31</number>
     </property>
    </widget>
   </item>
   <item row="7" column="2" colspan="2">
    <widget class="QSpinBox" name="fieldInitialDesignator">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>15</number>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="4">
    <widget class="QLabel" name="label_74">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;NOTE:&lt;/span&gt; Â  This team's species option will only be used if the &amp;quot;Player Species&amp;quot; option under &amp;quot;General Settings&amp;quot; is set to &amp;quot;Use Team Species.&amp;quot;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="4">
    <spacer name="verticalSpacer_15">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>285</width>
       <height>45</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
